Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/ScrollingObjectCollection.cs

```diff
@@ -0,0 +1,2198 @@
+        }
+
+        /// <summary>
+        /// The amount of time the user's hand can intersect a controller item before its considerd a drag gesture
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The amount of time the user's hand can intersect a controller item before its considerd a drag gesture")]
+        [Range(0.0f, 2.0f)]
+        private float dragTimeThreshold = 0.25f;
+
+        public float DragTimeThreshold
+        {
+            get => dragTimeThreshold;
+            set => dragTimeThreshold = value;
+        }
+
+        /// <summary>
+        /// Determines whether a near scroll gesture is released when the engaged fingertip is dragged outside of the viewable area.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Determines whether a near scroll gesture is released when the engaged fingertip is dragged outside of the viewable area.")]
+        private bool useNearScrollBoundary = false;
+
+        public bool UseNearScrollBoundary
+        {
+            get => useNearScrollBoundary;
+            set => useNearScrollBoundary = value;
+        }
+
+        /// <summary>
+        /// The direction in which content should scroll
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The direction in which content should scroll")]
+        private ScrollDirectionType scrollDirection;
+
+        public ScrollDirectionType ScrollDirection
+        {
+            get => scrollDirection;
+            set => scrollDirection = value;
+        }
+
+        /// <summary>
+        /// Toggles whether the <see cref="ScrollingObjectCollection"/> will use the Camera <see cref="Camera.onPreRender"/> event to hide items in the list. The fallback is <see cref"MonoBehaviour.LateUpdate()"/>.
+        /// </summary>
+        /// <remarks>This is especially helpful if you're trying to scroll dynamically created objects that may be added to the list after <see cref"MonoBehaviour.LateUpdate()"/> such as <see cref"MonoBehaviour.OnWillRender()"/></remarks>
+        [SerializeField]
+        [Tooltip("Toggles whether the scrollingObjectCollection will use the Camera OnPreRender event to hide items in the list")]
+        private bool useOnPreRender;
+
+        public bool UseOnPreRender
+        {
+            get => useOnPreRender;
+            set
+            {
+                if (cameraMethods == null)
+                {
+                    cameraMethods = CameraCache.Main.gameObject.EnsureComponent<CameraEventRouter>();
+                }
+
+                if (value)
+                {
+                    cameraMethods.OnCameraPreRender += OnCameraPreRender;
+                }
+                else
+                {
+                    cameraMethods.OnCameraPreRender -= OnCameraPreRender;
+                }
+
+                useOnPreRender = value;
+            }
+        }
+
+        /// <summary>
+        /// Amount of (extra) velocity to be applied to scroller.
```

Comment:
This exaggerates the calculated velocity in the scrolled list. The usage would be make a finger flick scroll the list in a disproportionally large amount. But I think you inadvertently found a bug as if this was zero it might kill all velocity - I need to check.